### ISSUE TITLE
Background (Theme, in  totality), Navbar and Categorization Changes.

### ISSUE THREAD

**Issue by simran212530** [ACM-IGDTUW/Open-Source-Challenge#108]
For #93 :
Hey, after a little bit of brainstorming and visiting the page multiple times, it gave me a few ideas that I would have to get implemented differently on the page, as a user. 

1.) The theme of the page can be white and blue, taking some light color of the background for eg, light blue/blue (I don't know web dev as of yet so in layman terms) and a little darker shade of the navigation bar. The black doesn't go with the logo as well.

2.) The navigation bar could be extended, like covering the entire width of the page and not just the right-hand side, with the ACM IGDTUW on top of that not along with those. To easily depict what I am trying to say I have attached another WordPress webpage screenshot, the navbar is can be similar to that. The font of the navigation is too simple, a little more beautiful font of that would be eye-catching, and a little bigger in size too. 

3.) The categorization of events would look great. As of now, when you visit the page, there is a lot of confusion, for eg. if I want to look up or register for an event to learn some specific programming language or when I want to go for an event making me learn app development, the events are mixed up and if they are even a larger in number it would take me a lot of time to select the field of my choice so that.

4.) The categorization would also use the space more efficiently which also makes the website look very clumsy right now. The optimization should also be implemented. For eg, in the current UI, when I scroll towards the end, there is only one event visible and the sides are completely empty, which doesn't look nice. 

Thanks. :") 
also, sorry for a very long issue.
![Screenshot (269)](https://user-images.githubusercontent.com/62848346/79591817-ae221200-80f6-11ea-913f-207ec3e0e756.png)
![Screenshot (270)](https://user-images.githubusercontent.com/62848346/79593531-39040c00-80f9-11ea-81ff-38579d7f2710.png)

**Comment by lisa761** [comment]
I agree with your changes, although I think you would be able to solve only some of it. Which one would you like to go for?

**Comment by simran212530** [comment]
Hey, now since it's optional, I would want to try as much as I can. :")

**Comment by lisa761** [comment]
Great then :) You also have a lot of time now ;)
